platform/android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -126,6 +129,39 @@ class FeatureContainer(context: Context, attrs: AttributeSet? = null) :
             .show()
     }
 
+    /**
+     * WebView uses Chrome under the hood
+     * An at least Chrome version 53 is required.
+     * Here, we check what Chrome version the user has installed,
+     * and show them a warning if it's lower or equals than 53 (android API 24).
+     */
+    private fun checkWebViewVersion() {
+        // userAgentString holds all the infor of User device, its format e.g.:
+        // """Mozilla/5.0 (Linux; Android 4.0.4; Galaxy Nexus Build/IMM76B)
+        // AppleWebKit/535.19 (KHTML, like Gecko)
+        // Chrome/18.0.1025.133 Mobile Safari/535.19"""
+        val userAgentString = webView.settings.userAgentString
+
+        val regex = """C(hrome|riOS)\/(?<major>\d+)[\d\.]""".toRegex()
+        val matchResult = regex.find(userAgentString)
+        val (_chrome, chromeVersion) = matchResult!!.destructured
+
+        println("ChromeVersion=$chromeVersion")
+        if (chromeVersion.toInt() <= 53) {
+            val message =
```

Comment:
This is user-facing, so I guess it should be translatable?

Reply:
yep it's tracked on my notes 🖕🏽 - but need a German hand here